ContactDetailsApi/V2/Gateways/Interfaces/ITenureDBGateway.cs

```diff
@@ -1,12 +1,12 @@
 using Hackney.Shared.Tenure.Domain;
-using System.Collections.Generic;
+using Hackney.Core.DynamoDb;
 using System.Threading.Tasks;
 
 namespace ContactDetailsApi.V2.Gateways.Interfaces
 {
     public interface ITenureDbGateway
     {
 
-        public Task<IEnumerable<TenureInformation>> GetAllTenures();
+        public Task<PagedResult<TenureInformation>> ScanTenures(string paginationToken, int? pageSize);
```

Comment:
Cant you pass the `ServicesoftFetchContactDetailsRequest` class instead?

Reply:
That would work, but best to keep the TenureDBGateway.ScanTenures gateway method decoupled from any Servicesoft request-specific functionality I think